config/locations.yml

```diff
@@ -103,165 +104,138 @@ covenantgn:
 covenantmh:   
   name: "Covenant Health - Misericordia Community Hospital"
   locationurl: "​https://www.covenanthealth.ca/corporate-information/library-services​"
-cross:   
-  name: "Alberta Health Services - Cross Cancer Institute"
-  locationurl: 
-cuc:   
-  name: "Burman University"
+cross:
+  name: "Cross Cancer Institute"
+  locationurl:
+cuc:
+  name: "Canadian University College"
   locationurl: "https://www.burmanu.ca/library"
-burman:   
-  name: "Burman University"
-  locationurl: "https://www.burmanu.ca/library"
-gprc:   
-  name: "Grande Prairie Regional College"
-  locationurl: "https://www.gprc.ab.ca/library"
-gprc_int:   
-  name: "Grande Prairie Regional College"
+gprcgp:
+  name: "Grande Prairie Regional College - Grande Prairie"
   locationurl: "https://www.gprc.ab.ca/library"
 gprcfrvw:   
   name: "Grande Prairie Regional College - Fairview"
   locationurl: "https://www.gprc.ab.ca/library"
-gprcgp:   
-  name: "Grande Prairie Regional College - Grande Prairie"
-  locationurl: "https://www.gprc.ab.ca/library"
 gprcint:   
   name: "Grande Prairie Regional College - Internet"
   locationurl: "https://www.gprc.ab.ca/library"
-grmacacc:   
-  name: "MacEwan University - Alberta College Campus"
+grmacacc:
+  name: "Grant MacEwan University - Alberta College Campus"
   locationurl: "https://library.macewan.ca/"
-grmaccac:   
-  name: "MacEwan University - Centre for Arts & Communications"
+grmaccac:
+  name: "Grant MacEwan University - Centre for Arts & Communications"
   locationurl: "https://library.macewan.ca/"
-grmacint:   
-  name: "MacEwan University - Internet"
+grmacint:
+  name: "Grant MacEwan University - Internet"
   locationurl: "https://library.macewan.ca/"
-grmacsoc:   
-  name: "MacEwan University - South CampusOn the MoveExpect Delay"
-  locationurl: "https://library.macewan.ca/​"
-grmactsv:   
-  name: "MacEwan University - Technical Services"
+grmactsv:
+  name: "Grant MacEwan University - Technical Services"
   locationurl: "https://library.macewan.ca/"
-grmacewan:   
-  name: "MacEwan University - City Centre Campus"
```

Comment:
https://www.neoslibraries.ca/member-libraries/macewan/ and https://library.macewan.ca/ appear to be branded "MacEwan".  Discovery has this as "Grant MacEwan." Which is correct?

Reply:
Good catch, @pgwillia, all MacEwan locations should be MacEwan University